src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -54,6 +52,18 @@ public void updateItem(TurboIssue issue, boolean empty) {
         this.setId(issue.getRepoId() + "_col" + parentPanelIndex + "_" + issue.getId());
     }
 
+    private void updateStyleForClosedIssues(TurboIssue issue){
```

Comment:
Yup, open issues do not require styling since it's *derived* from JavaFX source files for ListView. It would be pretty cumbersome to overwrite a lot of CSS associated with the default list view. I have refactored the code to check whether the open issues contain the closed styling. If so, it is removed. I have also renamed the function.